datamap/mv/core/src/test/scala/org/apache/carbondata/mv/rewrite/MVCreateTestCase.scala

```diff
@@ -0,0 +1,676 @@
+package org.apache.carbondata.mv.rewrite
```

Comment:
please add license.
license is missing in all testcase files

Reply:
ok